scripts/quests/otherAreas/A_Bitter_Past.lua

```diff
@@ -0,0 +1,154 @@
+-- ???        : !pos 58 -7 27
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.A_BITTER_PAST)
+
+quest.reward =
+{
+    item = xi.items.YINYANG_LORGNETTE,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Frescheque'] = quest:progressEvent(151),
+
+            onEventFinish =
+            {
+                [151] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        }
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Frescheque'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:hasKeyItem(xi.ki.TINY_WRISTLET) then
+                    else
+                        return quest:event(158)
+                    end
+                end,
+            },
+            ['Raminey'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 0 then
+                    return quest:event(152)
+                    end
+                end,
+            },
+            ['Equette'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:event(153)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [152] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 1)
+                end,
+
+                [153] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+
+                [158] = function(player, csid, option, npc)
+                    if quest:complete(player) then
+                        player:delKeyItem(xi.ki.TINY_WRISTLET)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.LUFAISE_MEADOWS] =
+        {
+            ['qm_bitter_past'] =
+            {
+                onTrigger = function(player, npc)
+                    local mobIDs =
+                    {
+                        16875775,
+                        16875776,
+                    }
+                    if quest:getVar(player, 'Prog') == 2 then
+                        npcUtil.popFromQM(player, npc, mobIDs, {claim = true})
```

Comment:
should be a pop message to return

Reply:
Wasn't any in the capture. so I'm setting the popFromQM as the return

Reply:
Found some proper dialogue in https://bitblt.org/elevyn/Sep2022/Zones/Lufaise%20Meadows/Dialog%20(EN).txt